### DISCUSSION TITLE
Can walletconnect dashboard not verify root domain?

### DISCUSSION THREAD

**Discussion by hamberluo** [WalletConnect/walletconnect-monorepo#2822] [Developer Support] [answered]
We use TXT with root domain for email postfix.
We want to verify subdomain by TXT.

**Comment by crypblizz8** [comment]
Hey @hamberluo , This is the current method we handle it. Does your domain provider not provide this option?

**Reply by hamberluo** [reply]
They do not provider his option.

**Comment by chadyj** [comment, marked as answer]
@hamberluo You can have multiple TXT records in your DNS configuration.
